capstone/src/main/java/com/google/sps/servlets/announcements/AnnouncementsSweeper.java

```diff
@@ -0,0 +1,48 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.common.collect.Streams;
+import java.time.Clock;
+import java.time.ZoneId;
+
+public class AnnouncementsSweeper {
+
+  private static ZoneId timeZone = ZoneId.of("America/Los_Angeles");
```

Comment:
Should we make this String a constant in Constants.java, so that it can be combined with mine after our PRs merge?